website/docs/getting-started/faq.mdx

```diff
@@ -0,0 +1,308 @@
+---
+id: faq
+title: FAQ
+position: 4
+---
+
+### How can I customize/style my form fields?
+
+You can style your form fields simply by passing a `className` property.
+
+### How can I create custom fields?
+
+You can create a custom field by wrapping your component inside the [`connectField`](/docs/api-helpers#connectfieldcomponent-options).
+
+The `connectField` will pass various props related to the form management, such as `onChange()` function, current field's value, errors and so on, to your component.
+
+Please visit the [Tutorials > Creating a custom field](/docs/tutorials-creating-custom-field) tutorial to see how to create your own fields.
+
+### How can I use a custom field in my form?
```

Comment:
This section can be merged with section above. Add link to the custom field example